test/powershell/engine/Help/HelpSystem.Tests.ps1

```diff
@@ -62,6 +62,16 @@ function RunTestCase
     }
 }
 
+Describe "Validate that <pshome>/<culture>/default.help.txt is present" -Tags @('CI') {
+
+    It "Get-Help Get-Help returns information about the help system." {
+
+        $helpContent = Get-Help Get-Help | Out-String
+        $helpContent | Should Match Get-Help
```

Comment:
I'd like to see something more explicit - is the goal here to see that help is rendered to a string, or is it the right string? I believe this assertion would be true even for `Get-Help Get-Command`. How about:
```
$HelpInfo = Get-Help Get-Help
$HelpInfo.Name | should be "Get-Help"
```
Is the requirement that Update-Help be found anywhere specific?